doc/new-guidelines.md

```diff
@@ -90,22 +133,49 @@ their styles in the top-level of the properties map, prefer to add them inside t
  ]
 ```
 
-### Always apply animated styles in the style file
+Also its fine to keep one liner styles in view
+
+```clojure
+;; ok
+[rn/view {:flex 1 :padding-top 5}]
```

Comment:
Please, add this style map inside a `:style` key in a map:

`{:style {:flex 1 :padding-top 5}}`

 just to make sure it's clear that we always want that `style` key.